packages/cli/src/commands/account/balance.ts

```diff
@@ -23,15 +25,27 @@ export default class Balance extends BaseCommand {
   ]
 
   async run() {
-    const kit = await this.getKit()
+    const client = await this.getPublicClient()
     const { args, flags } = await this.parse(Balance)
 
-    console.log('All balances expressed in units of 10^-18.')
-    printValueMap(await kit.getTotalBalance(args.arg1 as string))
+    console.log('All balances expressed in units of wei.')
+
+    // TODO this typing needs to be handled better...
+    printValueMapRecursive(
+      await getTotalBalance(client as any as PublicClient, args.arg1 as Address)
```

Comment:
nit: we have now `StrongAddress`, `Address` from celo/base and `Address` from viem, let's stick to celo/base for now no?